server/rest/src/main/java/org/infinispan/rest/resources/ServerResource.java

```diff
@@ -466,6 +473,30 @@ private CompletionStage<RestResponse> dataSourceTest(RestRequest restRequest) {
       }, invocationHelper.getExecutor());
    }
 
+   private CompletionStage<RestResponse> getCacheConfigDefaultAttributes(RestRequest request) {
+      Configuration configuration = new ConfigurationBuilder().build();
+      Map<String, String> attributes = new LinkedHashMap<>();
+      allAttributes(configuration, attributes);
+      return asJsonResponseFuture(Json.make(attributes), isPretty(request));
+   }
+
+   private static void allAttributes(ConfigurationElement<?> element, Map<String, String> attributes) {
+      for (Attribute<?> attribute : element.attributes().attributes()) {
+         AttributeDefinition<?> definition = attribute.getAttributeDefinition();
+         String value;
+         if(attribute.getInitialValue() == null){
```

Comment:
I would recommend importing the IntelliJ style code from https://github.com/infinispan/infinispan/tree/main/ide-settings so it will automatically fix formatting things such as this line.